AllHitAnalyze_New.py

```
@@ -1,162 +1,162 @@
+
+	def BarChart(self, axes:Axes) -> None:
+		# f(x) = \frac{1}{\sigma\sqrt{2\pi}} e^{-\frac{(x-\mu)^2}{2\sigma^2}}
+		# define PDF functions
+		def PDF(x:int) -> float:
+			return (E ** (((x - self.avg) ** 2) / (-2 * self.std ** 2))) / (((2 * PI) ** 0.5) * self.std) * self.summation;
+		def PDFEx(x:int) -> float:
+			return (E ** (((x - self.avgEx) ** 2) / (-2 * self.stdEx ** 2))) / (((2 * PI) ** 0.5) * self.stdEx) * self.summationEx;
+		def PDFEX(x:int) -> float:
+			return (E ** (((x - self.avgEX) ** 2) / (-2 * self.stdEX ** 2))) / (((2 * PI) ** 0.5) * self.stdEX) * self.summationEX;
+		# init chart argument
+		colors:list[str] = ['red','orange','yellow','green','cyan','blue','purple'];
```

Ruff (0.14.8)

Local variable `colors` is assigned to but never used

Remove assignment to unused variable `colors`

(F841)